Makefile.am

```diff
@@ -221,7 +221,7 @@ anaconda-release-build: anaconda-rpm-build
 
 anaconda-iso-creator-build:
 	$(CONTAINER_ENGINE) build \
-	$(CONTAINER_BUILD_ARGS) \
+	$(CONTAINER_BUILD_ARGS) --pull-always \
 	$(CONTAINER_ADD_ARGS) \
```

Comment:
I think I no longer understand why we have `CONTAINER_BUILD_ARGS` and `CONTAINER_ADD_ARGS`, and why this additional parameter isn't in one of these.

(I think this can be resolved later, we want the container now.)

Reply:
You can overload that easily by make call and also it's easier to avoid missing some build when adding arguments.